types/storybook__addon-storyshots/index.d.ts

```diff
@@ -52,8 +52,10 @@ export function renderWithOptions(options?: SnapshotOptions): Test;
 
 export function getSnapshotFileName(context: StoryContext): string;
 
-// tslint:disable-next-line no-unnecessary-generics
-export default function initStoryshots<Rendered>(options: InitOptions<Rendered>): void;
+export default function initStoryshots<Rendered>(
+  // tslint:disable-next-line no-unnecessary-generics
```

Comment:
Regarding this lint rule - is it possible to get rid of the generic type and make `options` be of type `InitOptions<any>`?

Reply:
Isn't it necessary to be able to pass the renderer's return type?

Reply:
Ah, I didn't see that test case.